addons/godot-firebase/database/reference.gd

```diff
@@ -66,14 +66,14 @@ func set_pusher(pusher_ref: HTTPRequest) -> void:
     if not _pusher:
         _pusher = pusher_ref
         add_child(_pusher)
-        _pusher.connect("request_completed", self, "on_push_request_complete")
+        _pusher.request_completed.connect(Callable(self, "on_push_request_complete"))
```

Comment:
I would rather see this as:

`_pusher.request_completed.connect(on_push_request_complete)`